docs: add instruction on how to use other modules in a module (#22081)
nuxt/nuxt: docs/2.guide/3.going-further/3.modules.md

})
```

> #### Using other modules in your module
> 
> If your module depends on other modules, you can add them by using `installModule`. For example, suppose you want to use `tailwindcss` framework in your module, you can add it as below:

Reviewer: ```suggestion
#### Using Other Modules in Your Module

If your module depends on other modules, you can add them by using Nuxt Kit's `installModule` utility. For example, if you wanted to use Tailwind CSS in your module, you could add it as below:
```

> ```js

Reviewer 2: ```suggestion
```ts
```
To hightlight types correctly

import { defineNuxtModule, createResolver, installModule } from '@nuxt/kit'

export default defineNuxtModule<ModuleOptions>({  

>   setup (options, nuxt) {

Reviewer 2: ```suggestion
  setup(options, nuxt) {
```

>   setup (options, nuxt) {

Reviewer 3: ```suggestion
  async setup (options, nuxt) {
```

    const { resolve } = createResolver(import.meta.url)

>     // We can inject our css file which includes tailwind's directives

Reviewer: ```suggestion
    // We can inject our CSS file which includes Tailwind's directives
```

>     nuxt.options.css.push(resolve( './runtime/assets/styles.css'))

Reviewer 2: ```suggestion
    nuxt.options.css.push(resolve('./runtime/assets/styles.css'))
```

>     installModule('@nuxtjs/tailwindcss', {

Reviewer 3: It is really important to always `await` on install module otherwise Nuxt lifecycle get's own of the order

```suggestion
    await installModule('@nuxtjs/tailwindcss', {
```

>       // module configurations

Reviewer 2: ```suggestion
      // module configuration
```

      exposeConfig: true,
      config: {
        darkMode: 'class',
        content: {
          files: [
            resolve('./runtime/components/**/*.{vue,mjs,ts}'),
            resolve('./runtime/*.{mjs,js,ts}')

>           ],

Reviewer: ```suggestion
          ]
```
